fashrevwall/run_streaming.py

```diff
@@ -0,0 +1,8 @@
+from TwitterClient import TwitterClient
```

Comment:
The streaming functionality doesn't ever end. You run it once and it keeps on streaming tweets until you switch it off. If you prefer to call it every X minutes instead, we better use the get_tweets_by_hashtag method from the TwitterClient. What do you think is better?